lib/nerves_network/if_supervisor.ex

```diff
@@ -26,7 +26,7 @@ defmodule Nerves.Network.IFSupervisor do
   end
 
   def setup(ifname, settings) do
-    pidname = pname(ifname)
+    _pidname = pname(ifname)
```

Comment:
Is this line needed at all anymore?  Or is it WIP?

Reply:
removed